docker/sorry-cypress/sorry-cypress.yml

```diff
@@ -0,0 +1,117 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Sorry Cypress Setup
+# ----------------------------------------------------------------------------------------------------------------------
+# this is the docker compose file for the sorry cypress setup
```

Comment:
I think this part of the comment can either be removed or should just reference the README. Currently it is redundant to the one above.

Reply:
Done (I just used the header from the other files)